cadc-rest/build.gradle

```diff
@@ -24,4 +24,7 @@ dependencies {
     compile 'org.opencadc:cadc-log:1.+'
 
     testCompile 'junit:junit:4.+'
+    testCompile 'org.mockito:mockito-core:2.+'
```

Comment:
I didn't notice this earlier.... is easymock dead? or no longer easy? :-)

I do not want mock testing used in these libraries - the tests end up too  brittle to be useful. Please remove.

Reply:
EasyMock uses `cglib`, which causes a whack of warnings regarding reflection, so it's been advised to move away from it.

Do we have a better way to test the `HttpServletRequest` requirements?

Reply:
Added stub class for `HttpServletRequest`.

Reply:
The real tests that are useful would be a dummy set of actions and testing in one or more servlet containers. Trying to mock or stub it out just adds another "servlet API impl" that may not be the same as tomcat or resin or whatever. That's why I don't think it is worth the effort to try to test code like this that is just a facade on the servlet API.